asreview/project/api.py

```diff
@@ -420,8 +420,10 @@ def add_review(
         if state is None:
             state = SQLiteState(fp_state)
             state.create_tables()
+            state.close()
         else:
             state.to_sql(fp_state)
+            state.close()
```

Comment:
Here you close a state object that was provided in the arguments. If I do `project.add_review(state)`, afterwards `state` is closed.

Reply:
Yeah thanks, nice side effect to find this bug as well. I changed the argument name.